gcp/values.yaml

```diff
@@ -0,0 +1,10 @@
+persistence:
+  enabled: true
+  storageClass: "standard"
+  size: "300Gi"
+
+storageClass:
+  defaultClass: true
+  name: env0-state-sc
```

Comment:
didn't we give our customers the ability to "variable"izing that one `env0-state-sc`?

Reply:
Yes we did. When we expose this one as a configurable Terraform module we would take it from there.  
This effort is just to refactor the current reference example.